go.mod

```diff
@@ -3,12 +3,10 @@ module cdap.io/cdap-operator
 go 1.12
 
 require (
-	github.com/coreos/etcd v3.3.15+incompatible // indirect
 	github.com/go-logr/logr v0.1.0
 	github.com/nsf/jsondiff v0.0.0-20190712045011-8443391ee9b6
 	github.com/onsi/ginkgo v1.11.0
 	github.com/onsi/gomega v1.8.1
-	github.com/stretchr/testify v1.4.0
```

Comment:
nit: are these changes needed?

Reply:
These changes were included because I ran `go mod tidy`, so they should no longer be necessary anymore.